packages/mediacenter/kodi-binary-addons/audiodecoder.usf/package.mk

```diff
@@ -31,4 +31,4 @@ PKG_LONGDESC="audiodecoder.usf"
 
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="kodi.audiodecoder"
-PKG_ADDON_PROJECTS="Generic Nvidia_Legacy RPi2 imx6 WeTek_Play"
+PKG_ADDON_PROJECTS="Amlogic Generic RPi2 WeTek_Play"
```

Comment:
Can remove WeTek_Play also

Reply:
no, because we haven't dropped 32-bit Amlogic hardware support in 9.0 so WP1 (which also supports WC) is still a valid build project. It can be removed in master after we branch 9.0.